src/test/java/seedu/address/network/NetworkTest.java

```diff
@@ -52,17 +56,18 @@ public void fetchAsStream_invalidWebsite_throwsIoexception() throws IOException
     @Test
     public void fetchAsString_success() {
         try {
-            String httpsContent = fetchAsString("https://se-education.org/");
+            String httpsContent = fetchAsString("https://cs2103-ay1819s2-w10-1.github.io/main/networktests/");
```

Comment:
This is good! Using a page that we have control of, nice!

Reply:
Danke!